frontend/src/components/elections/CreateElection.tsx

```diff
@@ -1,14 +1,26 @@
 "use client";
-import { ELECTION_FACTORY_ABI, getFactoryAddress } from "@/contracts/ElectionFactory";
+import {
+  ELECTION_FACTORY_ABI,
+  getFactoryAddress,
+} from "@/contracts/ElectionFactory";
 import { useSDK } from "@metamask/sdk-react";
 import { useRouter } from "next/navigation";
 import { SyntheticEvent, useState } from "react";
 import Web3 from "web3";
+import { DatePicker } from "@nextui-org/react";
```

Comment:
@Ratawine7 The package you are importing has not even been installed as a dependency.
I am checking the `frontend/package.json` file and it is not part of the list of deps.